include/seqan3/alphabet/nucleotide/dna4.hpp

```diff
@@ -260,6 +273,11 @@ struct dna4
         internal_type::UNKNOWN, internal_type::UNKNOWN, internal_type::UNKNOWN, internal_type::UNKNOWN,
         internal_type::UNKNOWN, internal_type::UNKNOWN, internal_type::UNKNOWN, internal_type::UNKNOWN  //255
     };
+public:
+    //!\privatesection
+    //!\brief The data member.
+    internal_type _value;
```

Comment:
Yes, private and protected **static** members are allowed in pod types and aggregate types.